mappings/net/minecraft/util/shape/OffsetVoxelShape.mapping

```diff
@@ -13,13 +13,16 @@ CLASS ckr net/minecraft/util/shape/VoxelShapeOffset
 		ARG 4 zMin
 		ARG 5 xMax
 		ARG 6 yMax
+		ARG 7 zMax
 	METHOD a set (IIIZZ)V
 		ARG 1 x
 		ARG 2 y
 		ARG 3 z
-		ARG 4 resize
+		ARG 4 aaa
```

Comment:
???

Reply:
Enigma glitched that I cannot rename these method (Most likely, enigma's change field targets a field of wrong index)

Reply:
Gotcha.